control/joy_controller/README.md

```diff
@@ -85,3 +85,25 @@
 | Autoware Disengage   | ○                          |
 | Vehicle Engage       | △                          |
 | Vehicle Disengage    | △                          |
+
+## XBOX Joystick Key Map
+
+| Action               | Button                |
+| -------------------- | --------------------- |
+| Acceleration         | RT                    |
+| Brake                | LT                    |
+| Steering             | Left Stick Left Right |
+| Shift up             | Cursor Up             |
+| Shift down           | Cursor Down           |
+| Shift Drive          | Cursor Left           |
+| Shift Reverse        | Cursor Right          |
+| Turn Signal Left     | LB                    |
+| Turn Signal Right    | RB                    |
+| Clear Turn Signal    | A                     |
+| Gate Mode            | B                     |
+| Emergency Stop       | View                  |
+| Clear Emergency Stop | Menu                  |
+| Autoware Engage      | X                     |
+| Autoware Disengage   | Y                     |
+| Vehicle Engage       | Left Trigger          |
+| Vehicle Disengage    | Right Trigger         |
```

Comment:
What is the difference between RT and Right Trigger? (Also LT and Left Trigger)?

Reply:
Right, it can be confusing, let's mark it as left/right stick button.

Reply:
Thank you, they become clearer meaning.